app/api/insert-dturn-statement.js

```diff
@@ -0,0 +1,59 @@
+// https://github.com/EnCiv/civil-pursuit/issues/163
+
+'use strict'
+
+import { insertStatementId } from '../dturn/dturn'
+
+import upsertPoint from './upsert-point'
+const { BSON } = require('bson')
+import { ObjectId } from 'mongodb'
+
+async function insertDturnStatement(dTurnId, pointObj, cb) {
+  // Anonymous functions to handle success/fail
+  const cbFailure = errorMsg => {
+    if (errorMsg) console.error(errorMsg)
+    if (cb) cb(undefined)
+  }
+  const cbSuccess = () => {
+    if (cb) cb(true)
+  }
+
+  // Verify arguments
+  if (arguments.length != 3) {
+    return console.error(`Expected 3 arguments (dTurnId, pointObj, cb) but got ${arguments.length}.`)
+  }
+
+  // Verify user is logged in.
+  if (!this.synuser || !this.synuser.id) {
+    return cbFailure('Cannot insert Dturn statement - user is not logged in.')
+  }
+
+  // Add userId to pointObj.
+  const userId = this.synuser.id
+  pointObj.userId = userId
+
+  // If _id is present, convert to BSON, else add ID.
+  const statementId = pointObj._id ? BSON.ObjectId.createFromHexString(pointObj._id.toString()) : new ObjectId()
+  pointObj._id = statementId
+
+  // Insert ID into dturn
+  const insertStatementResult = insertStatementId(dTurnId, userId, statementId)
+
+  // If insert to dturn failed - discussion isn't initialized, and don't need to try to insert into Points.
+  if (!insertStatementResult) {
+    return cbFailure()
+  }
+
+  // Attempt to insert pointObj.
+  const pointInsertCb = success => {
```

Comment:
It's mostly just a preference - I like pulling out longer arguments (objs/functions) so I can more easily focus on each part unnested when reading/changing things - if I need to reorder or replace the arguments I can look for and change a one-word variable as well.

Though I suppose there's not much difference in this case since the function's just a few lines and the expected  arguments for upsertPoint are set in stone - kind of just a habit.